homeassistant/helpers/trigger.py

```diff
@@ -621,3 +625,103 @@ async def async_get_all_descriptions(
 
     hass.data[TRIGGER_DESCRIPTION_CACHE] = new_descriptions_cache
     return new_descriptions_cache
+
+
+class TargetSelectorStateChangeTracker:
+    """Helper class to manage state change tracking for target selectors."""
```

Comment:
Same comment as for the class name
```suggestion
    """Helper class to manage state change tracking for targets."""
```